mne/decoding/transformer.py

```diff
@@ -245,6 +245,111 @@ def inverse_transform(self, X, y=None):
         return X.reshape(-1, self.n_channels, self.n_times)
 
 
+class Vectorizer(TransformerMixin):
```

Comment:
You have to add `@deprecated` (in mne.utils.deprecated IIRC), look across the package to see how to use it.